common/shlibs

```diff
@@ -22,6 +22,8 @@ libm.so.6 glibc-2.32_1
 libpthread.so.0 glibc-2.32_1
 librt.so.1 glibc-2.32_1
 libdl.so.2 glibc-2.32_1
+libm4ri-0.0.20200125.so m4ri-20200125_1
+libm4ri-0.0.20200125.so m4ri-devel-20200125_1
```

Comment:
Please remove this line: the soname is in the package "m4ri", not in the package "m4ri-devel".